examples/kustomization/operator-external-idp-oid/console-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: console
+  namespace: minio-operator
+spec:
+  template:
+    spec:
+      containers:
+        - name: console
+          env:
+          - name: CONSOLE_IDP_URL
+            value: https://myidpserver.com/realms/realmname/.well-known/openid-configuration
+          - name: CONSOLE_IDP_CLIENT_ID
+            value: "" # Client registered in Open ID
+          - name: CONSOLE_IDP_SECRET
+            value: "" #Client secret in Open ID
+          - name: CONSOLE_IDP_CALLBACK_DYNAMIC
+            value: "on"
+#          - name: CONSOLE_IDP_SCOPES
```

Comment:
Do we want to retain this commented block for reference?

Reply:
mmm, maybe not it is metioned in the readme, let me remove it